index.js

```diff
@@ -0,0 +1,19 @@
+const menu = document.querySelector('.menu');
+const closeMenu = document.querySelector('.close-item');
+const portfolioMenus = document.querySelector('.portfolio-menu');
+
+const showMenu = () => {
+  const mobileMenu = document.querySelector('.mobile-menu');
+  mobileMenu.style.display = 'flex';
+};
+
+const hideMenu = () => {
+  const mobileMenu = document.querySelector('.mobile-menu');
+  mobileMenu.style.display = 'none';
+};
+
+menu.addEventListener('click', showMenu);
+closeMenu.addEventListener('click', hideMenu);
+portfolioMenus.forEach((element) => {
+  element.addEventListener('click', hideMenu);
+});
```

Comment:
- [ ] Please fix the` addEventListner`, I can still see the menu after clicking on the links. see the requirement 👉 [ requirement for menu option](https://user-images.githubusercontent.com/61562626/127021523-e18a8d4f-9e97-4d12-80b8-d2bf4575216e.jpeg)

![menu2](https://user-images.githubusercontent.com/61562626/127021229-c6f9544f-918f-469b-982d-e4222ab87436.gif)